jcl/src/java.base/share/classes/java/lang/String.java

```diff
@@ -253,8 +253,26 @@ static void initCompressionFlag() {
 		}
 	}
 
-	static boolean compressible(char[] c, int start, int length) {
-		for (int i = start; i < length; ++i) {
+	/**
+	 * Determines whether the input character array can be encoded as a compact
+	 * Latin1 string.
+	 *
+	 * <p>This API implicitly assumes the following:
+	 * <blockquote><pre>
+	 *     - {@code length >= 0}
+	 *     - {@code start >= 0}
+	 *     - {@code start + length <= data.length}
+	 * <blockquote><pre>
+	 *
+	 * @param c      the array of characters to check
+	 * @param start  the starting offset in the character array
+	 * @param length the number of characters to check starting at \c start
+	 *
+	 * @throws IndexOutOfBoundsException
+	 *          when {@code length < 0, start < 0} or {@code start + length > data.length}
+	 */
```

Comment:
Just realized I left this copy/paste here. I need to remove this. I'll take care of this tomorrow morning.

Reply:
Fixed in [force push](https://github.com/eclipse/openj9/compare/e30bce09fd5431e04295e9398b2c54709f146403..b5085c04868181f1dfd2c4bc7e932ea9c2c364d3)